suite/tests/api/drdecode_aarch64.c

```diff
@@ -130,6 +130,69 @@ test_noalloc(void)
      */
 }
 
+static void
+test_categories(void)
+{
+    const uint raw[] = {
+        0x00000000, /* no category */
+        0x12020000, /* int math */
```

Comment:
Add the disassembly string in the comment for each so a human can understand this.